apps/explorer/src/ui/SenderRecipientAddress.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva } from 'class-variance-authority';
+
+import { ReactComponent as Recipient } from './icons/checkfill.svg';
+import { ReactComponent as Sender } from './icons/sender.svg';
+
+import { Link } from '~/ui/Link';
+
+const senderRecipientAddressStyles = cva(
+    ['break-all flex flex-row gap-2 items-center'],
+    {
+        variants: {
+            isCoinTransfer: {
+                true: 'ml-6',
```

Comment:
Spacing shouldn't be in this component, so this variant is probably not needed at all.